metadata-ingestion/src/datahub/ingestion/source/powerbi/config.py

```diff
@@ -130,21 +148,27 @@ class PowerBiAPIConfig(EnvBasedSourceConfigBase):
     )
     # Enable/Disable extracting ownership information of Dashboard
     extract_ownership: bool = pydantic.Field(
-        default=True, description="Whether ownership should be ingested"
+        default=True,
+        description="Whether ownership should be ingested. Admin access should be given to the configured "
+        "Azure AD Application ",
     )
     # Enable/Disable extracting report information
     extract_reports: bool = pydantic.Field(
         default=True, description="Whether reports should be ingested"
     )
     # Enable/Disable extracting lineage information of PowerBI Dataset
     extract_lineage: bool = pydantic.Field(
-        default=True, description="Whether lineage should be ingested"
+        default=True,
+        description="Whether lineage should be ingested. Admin access should be given to the configured Azure AD "
```

Comment:
"Whether lineage should be ingested between X and Y. Admin API access is required if this setting is enabled".

Reply:
done